### ISSUE TITLE
Decal 2.9.8.0 compatibility

### ISSUE THREAD

**Issue by ghostchillidog** [harliq/AceCreator#18]
I've installed Decal 2.9.8.0 and noticed that AceCreator didn't show up in game.
I removed it and attempted to re-add the DLL to Decal and got the error "failed to register plugin".

Uninstalling 2.9.8.0 and reinstalling 2.9.7.5 allowed me to re-add the AceCreator DLL and it started working again in game.

**Comment by Nicon4454** [comment]
i have the same issue but it is for 2.9.8.3 of decal

**Comment by jadaire** [comment]
I was hoping for a fix. I just started a new server!

**Comment by Arcanehand** [comment]
Yeah there still seems to be zero fix for this.  Wish I had known this before I bothered to update decal that this was going to be a major issue.
